CHANGELOG.md

```diff
@@ -1,4 +1,8 @@
-### [0.15.1](https://github.com/eclipse/che-che4z-lsp-for-hlasm/compare/0.15.0...0.15.1) (2021-11-11)
+# HLASM Language Support Changelog
```

Comment:
This file is maintained by the CI, we can try adding headers and so on, but I'd prefer doing in later on to verify it actually works correctly.